### PR TITLE
make able read paper from folder

### PR DESCRIPTION
**Pull request by hack-wrench**
## About The Pull Request

Improved due to the complexity of running a bureaucracy (I don't have enough GBP on TG for this QoL)

## How This Contributes To The Nova Sector Roleplay Experience

Conveniently store photos and papers in a folder without having to reach in and disorder them

## Changelog

:cl:
qol: make able reading the paper in the folder
/:cl:

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -367,7 +367,7 @@
 		return UI_CLOSE
 	if(!user.can_read(src))
 		return UI_CLOSE
-	if(in_contents_of(/obj/machinery/door/airlock) || in_contents_of(/obj/item/clipboard))
+	if(in_contents_of(/obj/machinery/door/airlock) || in_contents_of(/obj/item/clipboard) || in_contents_of(/obj/item/folder))
```
Comment (SomeRandomOwl) [inline code/modules/paperwork/paper.dm:370]:
```suggestion
	if(in_contents_of(/obj/machinery/door/airlock) || in_contents_of(/obj/item/clipboard) || in_contents_of(/obj/item/folder)) // NOVA EDIT - ORIGINAL: if(in_contents_of(/obj/machinery/door/airlock) || in_contents_of(/obj/item/clipboard))
```

**Review by SomeRandomOwl — CHANGES_REQUESTED** [review]

**Comment by OrionTheFox** [comment]
This kind of small QoL should be upstreamed. Semi-modular code improvements should just be fully integrated upstream code improvements.

**Comment by hack-wrench** [comment]
> This kind of small QoL should be upstreamed. Semi-modular code improvements should just be fully integrated upstream code improvements.

Upstream rejected QoL due to negative GBP

**Comment by hack-wrench** [comment]
> Would you mind if I tried remaking your PR upstream to see if they'd take it? In the meantime I'm fine with this

No problem

**Pull request merged**